packages/nocodb/src/lib/services/auditService.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  AuditRowUpdatePayloadType,
+  CommentRowPayloadType,
+} from 'nocodb-sdk/build/main/lib/CustomAPI';
```

Comment:
can we do `import type { xxx } from 'nocodb-sdk` instead? apply to all services.

Reply:
Yes we can, will add rule in eslint - https://typescript-eslint.io/rules/consistent-type-imports/

Reply:
Will do it in a different PR, commit reference - https://github.com/nocodb/nocodb/commit/37ed4048a5ad7de06e303e56652642bd6cf35b9e